types/rebass/rebass-tests.tsx

```diff
@@ -267,3 +271,81 @@ const AsProp = () => (
         <Flex as="input">Flex as input</Flex>
     </Box>
 )
+
+interface Theme {
+    dark: {
+        backgroundColor: string
+    }
+}
+
+const WithTypedTheme = () => {
+    const {Box} = rebassComponents as RebassComponentsModule<Theme>
+    return (
+        <Box theme={{dark: {backgroundColor: '#666666'}}} />
```

Comment:
Hmm.. Not sure i follow the use case here. According to styled components documentation, theming is done by using a `<ThemeProvider theme={theme}>...</ThemeProvider>`, not sure why one would want to one-off theme components like this. You got some example use case 😁?. Check it out https://rebassjs.org/theming